### ISSUE TITLE
[Bug] Saving issue

### ISSUE THREAD

**Issue by Dasyat1s** [MeteorDevelopment/meteor-client#1707]
### Describe the bug

the settings are saving on all the exisiting configs somehow 

### Steps to reproduce

```bash
load a config, don't save it, open another config and some modules config will transfer from a config to another (hole esp, autotrap, ca and some other random stuff)
```

### Meteor Version

1388

### Minecraft Version

1.17.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by RedCarlos26** [comment]
Duplicata of #1606 @arlomcwalter

**Comment by MineGame159** [comment]
Fixed in 4316e128240021d35058dde7f0d6ad2b0b8c14b5.
